core/server/master/src/main/java/alluxio/master/file/meta/UfsSyncPathCache.java

```diff
@@ -1,211 +0,0 @@
-/*
```

Comment:
for the ease of feature rollout, do you think if we need to create a SyncPathCache interface with the existing implementation and your new one and have a feature flag (property key) to switch around?

Reply:
I worry this would over complicate things, can check with @yuzhu

Reply:
yea i'm a bit indifferent and will let @yuzhu make the call.